2.4/root/usr/bin/run-mongod

```diff
@@ -15,6 +15,7 @@ export MONGODB_DATADIR=/var/lib/mongodb/data
 export MONGODB_NOPREALLOC=${MONGODB_NOPREALLOC:-true}
 export MONGODB_SMALLFILES=${MONGODB_SMALLFILES:-true}
 export MONGODB_QUIET=${MONGODB_QUIET:-true}
+export MONGODB_TEXT_SEARCH_ENABLE=${MONGODB_TEXT_SEARCH_ENABLE:-false}
```

Comment:
is it ENABLED or ENABLE?  doc says ENABLED, this says ENABLE
(should be ENABLED imho)